thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -415,7 +415,8 @@ public Response updateAnomalyFunction(@NotNull @PathParam("id") Long id,
       @QueryParam("exploreDimension") String exploreDimensions,
       @QueryParam("filters") String filters,
       @QueryParam("properties") String properties,
-      @QueryParam("isActive") Boolean isActive) throws Exception {
+      @QueryParam("isActive") Boolean isActive,
+      @QueryParam("frequency") String frequency) throws Exception {
```

Comment:
is it frequency of anomaly job or granularity of data ?
if later then please rename it to something like aggGranularity (aggregation granularity)